src/MovieCard/MovieCard.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 import '../MovieCard/MovieCard.css'
 
-const MovieCard = ({movieCard}) => {
+const MovieCard = ({movieCard, toggleMovie}) => {
   return (
-    <article className="movie-card">
+    <article className="movie-card" onClick={() => toggleMovie(movieCard.id)}>
       <img  className="movie-image" src={movieCard.poster_path} alt=""/>
        <h4 className="movie-title"> {movieCard.title}</h4>
```

Comment:
Are there h2's and h3's on the page? If not, you shouldn't skip them and jump to an H4. Headings shouldn't skip levels. That's really confusing for users using a screenreader. If you're choosing an h4 for it's size, just remember that you can use any element and size it using css